plugins/modules/firewalld.py

```diff
@@ -84,6 +84,13 @@
     description:
       - Whether to run this module even when firewalld is offline.
     type: bool
+  target:
+    description:
+      - firewalld Zone target
+      - If state is set to C(absent), this will reset the target to default
+    choices: [ default, ACCEPT, DROP, REJECT ]
+    type: str
+    version_added: 0.2.0
```

Comment:
Latest released version is 1.1.1 according to https://github.com/ansible-collections/ansible.posix/tags, but I'm not sure what would be next version (1.2.0 ? 2.0.0 ? something else ?)

Maybe 1.2.0, but I'm not sure.

Reply:
Since we are adding some functionality I would say 1.2.0